README.md

```diff
@@ -171,8 +172,10 @@ PRs welcomed!
 - [simple-poem-typst](https://github.com/asibahi/simple-poem-typst) - An application of the `measure` function to set Arabic poetry.
 - [typst-ansi_render](https://github.com/8LWXpg/typst-ansi_render) - A library to render text with ANSI escape sequences
 - [typst-boxes](https://github.com/lkoehl/typst-boxes) - A library to draw colorful boxes.
+- [showybox]([https://github.com/lkoehl/typst-showybox](https://github.com/Pablo-Gonzalez-Calderon/showybox-package)) - A Typst package for creating colorful and customizable boxes.
```

Comment:
The first URL gives me a 404 page not found error:

[lkoehl/typst-showybox](https://github.com/lkoehl/typst-showybox)

but the second one works:

[Pablo-Gonzalez-Calderon/showybox-package](https://github.com/Pablo-Gonzalez-Calderon/showybox-package)

Could you please remove the broken link, leaving in only `Pablo-Gonzalez-Calderon/showybox-package`? Thanks!